### PR TITLE
Removed mapping screen.

### PR DESCRIPTION
**Pull request by yunikkk**
Android part of #1744

### CODE REVIEW

File: common/src/main/res/layout/activity_replay.xml [hunk 1]
```diff
@@ -0,0 +1,446 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (yunikkk) [inline common/src/main/res/layout/activity_replay.xml]: Layouts for replay and teaser are more different now, decided to add separate layout for now, since soon we'll unite all three apps together.

File: gradle/versions.gradle [hunk 2]
```diff
@@ -15,8 +15,8 @@ ext {
     android_x_multidex = '2.0.1'
 
     vision = '0.11.0'
-    teaser_version_name = '0.10.1'
-    teaser_version_code = 20
+    teaser_version_name = '0.11.0'
```
Comment (kiryldz) [inline gradle/versions.gradle]: Shouldn't we do that in separate PR?

Comment (yunikkk) [reply gradle/versions.gradle]: Well, oook

Comment (yunikkk) [reply gradle/versions.gradle]: Removed
